maas/plugins/horizon_check.py

```diff
@@ -111,5 +111,8 @@ def main(args):
         parser.add_argument('ip',
                             type=ipaddr.IPv4Address,
                             help='horizon dashboard IP address')
+        parser.add_argument('site_name_regexp',
```

Comment:
I'm not seeing where this variable is getting a value in this PR. Is it meant to be there for overides, but not set by default?